Documentation/requirements.md

```diff
@@ -0,0 +1,84 @@
+# Requirements
+
+For the most part the codebase is self-contained (e.g. all dependencies are vendored), but assembly of the stage1 requires some other tools to be installed on the system.
+
+## Build-time Requirements
+
+### Basic
+
+* GNU Make
+* Go 1.4+ (ideally 1.5.3 or later)
+* autoconf
+* aclocal (usually a part of automake)
+* bash
+* git
+* glibc
+  * development headers
+  * the rkt binary links against the library
+* gofmt (usually distributed with Go)
+* govet (usually distributed with Go)
+* TrouSerS (only when TPM is enabled)
+  * development headers
+  * the rkt binary links against the library
+* gpg (when running functional tests)
+
+### Additional requirements when also building any stage1 image flavor
+
+* glibc
+  * development headers
+  * the stage1 binaries link against the static library
```

Comment:
I prefer to use the "static library" (or "statically-linked library"). I never heard about "static archive" before. I'd say that static libraries being archives of object files is a kind of implementation detail.